platform/externalstoragemigration/content_uri_update_script.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+red=$(tput setaf 1)
+green=$(tput setaf 2)
+yellow=$(tput setaf 3)
+default=$(tput sgr0)
+
+STORAGETYPE=$1
+HOST=$2
+USERNAME=$3
+PASSOWORD=$4
+DBNAME=$5
+
+
+
+check_input_value() {
+  if [ -z "$2" ];
+  then
+    echo "$red $(date) $1 is blank or not provided $default"
+    exit 1;
+  elif [ "$1" == "STORAGETYPE" ];
+  then
+    if [[ "$2" != "azure" && "$2" != "s3" ]];
+    then
+    echo "$red $(date) $1 allowed values are azure or s3 $default"
+    exit 1;
+    fi
+  else
+    echo "$green $(date) $1 is provided $default"
+  fi
+}
+
+
+# function to validate database connection
+validate_dbconn() {
+   
+    echo "$green Validating connection to $HOST $default"
+
+    result=$(sqlcmd -S ${HOST} -U ${USERNAME} -P ${PASSOWORD} -d ${DBNAME} -Q "SELECT DB_NAME()" )
```

Comment:
check if sqlcmd and other dependency are already installed on the system

Reply:
done